### ISSUE TITLE
Issue: "JFrog's Free-Tier subscription has been deactivated."

### ISSUE THREAD

**Issue by KarlStraussberger** [gerbera/gerbera#2988]
### Discussed in https://github.com/gerbera/gerbera/discussions/2967

<div type='discussions-op-text'>

<sup>Originally posted by **udippel** January 14, 2024</sup>
... this is what I get when I want to download the most recent build at the usual URL.
Any changes?</div>

**Comment by nmanolov** [comment]
`apt-get update` gets an error for me:
```Get:5 https://gerbera.jfrog.io/artifactory/debian jammy InRelease [11.5 kB]
Err:5 https://gerbera.jfrog.io/artifactory/debian jammy InRelease
  Clearsigned file isn't valid, got 'NOSPLIT' (does the network require authentication?)
Reading package lists... Done     
E: Failed to fetch https://gerbera.jfrog.io/artifactory/debian/dists/jammy/InRelease  Clearsigned file isn't valid, got 'NOSPLIT' (does the network require authentication?)
E: The repository 'https://gerbera.jfrog.io/artifactory/debian jammy InRelease' is no longer signed.
```

It is caused by:
```
curl https://gerbera.jfrog.io/artifactory/debian/dists/jammy/InRelease
<html>
<head><title>302 Found</title></head>
<body>
<center><h1>302 Found</h1></center>
<hr><center>nginx</center>
</body>
</html>
```
When I hit the URL in the browser I also get the title of the issue.

**Comment by Arthur-Milchior** [comment]
Can I suggest editing the documentation until this is repaired or an alternative is found.
This would avoid 

```
> curl -fsSL https://gerbera.jfrog.io/artifactory/api/gpg/key/public | sudo apt-key add -
gpg: no valid OpenPGP data found.
```

As I just did.

I came here to report the same issue, and now understand why it occurred, thanks

**Comment by whyman** [comment]
So I spent some time looking at getting hosted on https://fury.io but it seems like they dont have good support for multiple OS versions. We have a lot of different versions and two OS variants to support so thats not ideal.

I think likely we will have to host something ourself, will look at some cloud free tier bucket storage options.

**Comment by nijel** [comment]
I know some projects using https://packager.io/, but I have no personal experience with it.

**Comment by udippel** [comment]
Do I read this correctly, that we've been out of packaged, installable candidates (.deb) for almost a quarter now? 
In case there is no solution in sight, l[https://docs.gerbera.io/en/latest/install.html](url) should be modified. 

In case someone has a complete development environment, I'd be happy to receive anything fresher than 2.0.0-7 as .deb

**Comment by whyman** [comment]
Yes, unfortunately we are all volunteers and do not have unlimited time :smiley: 

The deb files are still being built in the CI and you can grab them from there directly, if you just want a more modern build in the mean time, or you can use other packaging such as the docker images.

You can see the assets that were built [here](https://github.com/gerbera/gerbera/actions/runs/8596093897)

I have also updated the release for 2.1.0, with deb files from that build.

**Comment by whyman** [comment]
Found some time this evening, can someone test out these docs? https://github.com/gerbera/gerbera/pull/3050/files

**Comment by Martii** [comment]
@whyman

> I have also updated the release for 2.1.0, with deb files from that build ([here](https://github.com/gerbera/gerbera/releases/tag/v2.1.0))

Bless you.

> can someone test out these docs? 

See pull request comments and:

<a href="https://github.com/gerbera/gerbera/assets/114709/5a505b3e-1b32-4a14-b2c4-bc2d4e6acd4e"><img src="https://github.com/gerbera/gerbera/assets/114709/5a505b3e-1b32-4a14-b2c4-bc2d4e6acd4e" width="192" height="105" alt="git repo redacted" title="Click to enlarge... git repo redacted"></a>

Need more simple testing I'm there for it.

Again... awesome and Thank you!

**Comment by udippel** [comment]
Yup, tried the 2.1.0. Thanks a bunch!

I know all too well how it feels as a volunteer, with family & stuff on the side ... . Still, I think it was about time to ask if this was like left unhandled.

**Comment by KarlStraussberger** [comment]
> Yup, tried the 2.1.0. Thanks a bunch!
> 
> I know all too well how it feels as a volunteer, with family & stuff on the side ... . Still, I think it was about time to ask if this was like left unhandled.

It's not unhandled anymore but with no replacement available, the text on the web page would not change the packager issue on your system.

**Comment by udippel** [comment]
Love that screenshot of Martii. Addresses partially an issue I had discussed with KarlStraussberger sometime in summer last year. While I know NOW, the beginner can't be expected to be aware of the config choices and consequences. 

I know that this doesn't really belong here, rather in Discussions, so I won't go into detail. If anybody wanted, we could continue over there. Though it shouldn't clutter the view statically, rather pop up at e.g. at a first run.
Just wanted to give out a 'thumb up!' for starting work on the issue.

**Comment by udippel** [comment]
e.g. here: https://github.com/gerbera/gerbera/discussions/2864#discussioncomment-6955517

Maybe earlier and later, too. 
I'd love to continue on this, since to me it is a major issue w.r.t. initial acceptance and user retention of this software. The features are aplenty, the web interface is well done. Though the user guidance especially for new users is almost not-existing, except one wanted to study the complete documentation in advance. 
(Read some of my discussion items at starting with gerbera.)

I know that architectural decisions would be necessary to overcome the partial overlap of configuration in a file (config.xml) and in the database. It is impossible at the moment to change from one to the other in flight. 

(Only to help understanding: like in any WAP. You configure, try out, eventually revert. AND you can save the configuration as a backup file. AFAIK, config in the gerbera database would be lost with the database. One might as well - if one had too much time on one's hand - write an exporter for the config items that writes to a file. And can read back from a file into the database. 
Unfortunately I have no more students that I could ask to do stuff like this for their thesis since I have retired.)

Therefore, the initial information and decision by the user is of relevance, which type he wants to follow; as indicated in your screen design. 
It had taken me a dozen of starts from scratch to know where I was. "The usual way of working with gerbera is: Configure your autoscan and let it observe your media folders." How would one know? 
Initial guidance could be most useful indeed!

**Comment by whyman** [comment]
Closed via #3050 and #3048
